web/themes/custom/votegov/templates/layout/page--node--voter-guide.html.twig

```diff
@@ -39,7 +39,7 @@
       </div>
     </div>
   </main>
-  <div role="region" aria-labelledby="voter-guide-cards">
+  <div role="region" aria-labelledby="{{ 'Know your voting rights' | t }}">
```

Comment:
this needs to be a `aria-label` attribute. `aria-lablledby` expects an id name.

Reply:
updated to `aria-label` did some more reading of the docs on MDN and think i have a better understanding now of the use between the 2